presto-iceberg/src/main/java/com/facebook/presto/iceberg/changelog/ChangelogOperation.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.changelog;
+
+import com.facebook.presto.spi.PrestoException;
+
+import static com.facebook.presto.spi.StandardErrorCode.NOT_SUPPORTED;
+
+public enum ChangelogOperation
+{
+    INSERT,
+    DELETE,
+    UPDATE_BEFORE,
+    UPDATE_AFTER;
+
+    public static ChangelogOperation fromIcebergChangelogOperation(org.apache.iceberg.ChangelogOperation operation)
```

Comment:
static import

Reply:
@yingsu00 We have to use the fully qualified class name since we have used the same enum name in Presto. Please let me know in case you are talking about something else.